lib/team_city_formatter/formatter.rb

```diff
@@ -104,20 +108,24 @@ def before_scenario_outline(cuke_scenario_outline)
         x.name = "#{cuke_scenario_outline.keyword}: #{cuke_scenario_outline.name}"
         x.example_column_names = cuke_example_rows.first.send(:data).keys
         x.examples =
-          cuke_example_rows.map do |example_row|
-            example_row.send(:data).values
-          end.map do |example_column_values|
-            Example.new.tap do |x|
-              x.column_values = example_column_values
+            cuke_example_rows.map do |example_row|
+              example_row.send(:data).values
+            end.map do |example_column_values|
+              Example.new.tap do |x|
+                x.column_values = example_column_values
+              end
```

Comment:
@gricsi it looks like these lines were simply indented another couple of spaces. Could you please revert the change here.